docs/sql-migration-guide.md

```diff
@@ -344,6 +344,12 @@ license: |
 
   - Since Spark 2.4.5, `TRUNCATE TABLE` command tries to set back original permission and ACLs during re-creating the table/partition paths. To restore the behaviour of earlier versions, set `spark.sql.truncateTable.ignorePermissionAcl.enabled` to `true`.
 
+  - Since Spark 2.4.5, `spark.sql.legacy.mssqlserver.numericMapping.enabled` configuration is added in order to support the legacy MsSQLServer dialect mapping behavior using IntegerType and DoubleType for SMALLINT and REAL JDBC types, respectively. To restore the behaviour of 2.4.3 and earlier versions, set `spark.sql.legacy.mssqlserver.numericMapping.enabled` to `true`.
+
+## Upgrading from Spark SQL 2.4.3 to 2.4.4
```

Comment:
As 2.4.4 is already out, this doc is in 2.4.5? So maybe it should be 2.4.3 -> 2.4.5?

Reply:
IMO, although it's late for 2.4.4, `2.4.3` to `2.4.4` will be correct. When the users upgrade from 1.6.3 to 3.0.0, they need to see all previous migration guides. If there is some regression on 2.4.5, the users can use 2.4.4 instead of 2.4.5.

Reply:
For `2.4.4` release doc, I'll update `spark-website` repository.

Reply:
ok sounds good. Thanks!

Reply:
Thanks!